docs/threats.md

```diff
@@ -0,0 +1,38 @@
+
+## Cylc-8 Threat Modelling Assessment outcomes
+
+(email from BOM staff to HO, 25 March 2019)
+
+> Last Thursday we held a security threat modelling session on Cylc-8 using the
+current architecture diagram you have provided. The result was the set of
+"user stories" we think need to be considered, which are listed below.  Most of
+them are "negative" user stories ie. they are stories we would like Cylc-8 to
+stop from happening.  User stories generally spark further conversations,
+particularly when it comes to someone trying to work out what to implement and
+how. I expect this set of stories will probably do the same!
+ 
+> The other key outcome of the meeting was that we confirmed that JupyterHub is
+on our Enterprise Architecture team's technology roadmap, which means it is an
+accepted technology for use at the BOM.
+
+> Security User Stories
+>
+- As an attacker I want to stop user A suite A from running
+- As an attacker I want to be able to fool authentication on the Cylc HUB
+- As an attacker I want to harvest usernames and passwords
+- As an attacker I want to be able to get the network map relevant to the Cylc solution
+- As an attacker I want to hijack current tokens to take over sessions
+- As an attacker I want to harvest suite state via the HTTP proxy on Cylc hub
+- As an attacker I want to execute my own suite/arbitrary code on HPC
```

Comment:
And/or other site infrastructure?

Reply:
(this should be in our response, rather than a change to the original questions)